hicommonwealth/commonwealth
# Added manual PWA install instructions for android

## Link to Issue
Closes: https://github.com/hicommonwealth/commonwealth/issues/6937

## Description of Changes
Added manual PWA install instructions for android, these will only show if the logic behind `Install` button returns some errors

## "How We Fixed It"
N/A

## Test Plan
- Important: test on a real android chrome on a real device, testing on ios chrome will not give the same prompt.
- Open any page, other than the landing page i.e `/`
- Verify you see the PWA prompt
- Click on the `Install` button (this as of now fails)
- Verify you see the help text showup in the prompt (it will showup with a 1000ms delay)

## Deployment Plan
N/A

## Other Considerations
N/A
